resources/packs/gcp/gcp.lr

```diff
@@ -794,7 +796,7 @@ private gcp.dns.policy {
   enableLogging bool
 }
 
-// GKE Cluster
+// GCP GKE Cluster
 private gcp.project.cluster @defaults("name") {
```

Comment:
@tas50 should we rename `gcp.project.cluster` to `gcp.project.gke`?

Reply:
I guess the suggestion here is to access the clusters like the other resources I added - `gcp.project.gke.clusters`?

Reply:
We change this in another PR since it is not related to this PR